var/spack/repos/builtin/packages/py-cmake-format/package.py

```diff
@@ -14,4 +14,5 @@ class PyCmakeFormat(PythonPackage):
 
     version('0.4.5', sha256='16602408c774cd989ecfa25883de4c2dbac937e3890b735be4aab76f9647875a')
 
-    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-pyyaml', type=('run',))
```

Comment:
```suggestion
    depends_on('py-pyyaml', type=('run'))
```

Reply:
Then the parenthesis should also be removed…